compiler/builtins/object.ts

```diff
@@ -84,17 +84,28 @@ export const __Object_values = (obj: any): any[] => {
       if (!Porffor.object.isEnumerable(ptr)) continue;
 
       let val: any;
-      Porffor.wasm`local ptr32 i32
-local.get ${ptr}
-i32.to_u
-local.tee ptr32
 
-f64.load 0 8
-local.set ${val}
-
-local.get ptr32
-i32.load8_u 0 17
-local.set ${val+1}`;
+      const tail: i32 = Porffor.wasm.i32.load16_u(ptr, 0, 16);
+      if (tail & 0b0001) {
+        const get: Function = __Porffor_object_accessorGet(ptr);
+
+        if (Porffor.wasm`local.get ${get}` != 0) {
+          val = get.call(obj);
+        }
+      } else {
+        Porffor.wasm`local ptr32 i32
+          local.get ${ptr}
+          i32.to_u
+          local.tee ptr32
+
+          f64.load 0 8
+          local.set ${val}
+
+          local.get ptr32
```

Comment:
The type is actually in the tail already in the upper 4 bits, could you use that instead of doing another read here? Thanks!